core/templates/services/user-backend-api.service.ts

```diff
@@ -141,11 +141,9 @@ export class UserBackendApiService {
   }
 
   async updatePreferencesDataAsync(
-      updateType: string,
```

Comment:
What about the call sites of this function, you will need to modify those.

Reply:
done